central/clusters/zip/render_test.go

```diff
@@ -68,11 +68,15 @@ func doTestRenderOpenshif(t *testing.T, clusterType storage.ClusterType) {
 	}
 	assertScannerRemote := func(obj runtime.Object) {
 		ds := obj.(*v1.DaemonSet)
-		assert.Len(t, ds.Spec.Template.Spec.Containers, 3)
-		mainImage := ds.Spec.Template.Spec.Containers[1].Image
-		nodeInvCont := ds.Spec.Template.Spec.Containers[2]
-		expectedScannerParts := strings.Split(strings.ReplaceAll(mainImage, "/main:", "/scanner-slim:"), ":")
-		assert.Truef(t, strings.HasPrefix(nodeInvCont.Image, expectedScannerParts[0]), "scanner-slim image (%q) should be from the same registry as main (%q)", nodeInvCont.Image, mainImage)
+		if clusterType == storage.ClusterType_OPENSHIFT4_CLUSTER {
+			assert.Len(t, ds.Spec.Template.Spec.Containers, 3)
+			mainImage := ds.Spec.Template.Spec.Containers[1].Image
+			nodeInvCont := ds.Spec.Template.Spec.Containers[2]
```

Comment:
Can we be sure that node inventory will always be at index position `2`?

Reply:
This is fixed in the template, so if someone would change that, this test would fail.
Let me make it a bit clearer and make an additional assertion that the 3rd container must be named `node-inventory`

Reply:
Done in [56b7b0d](https://github.com/stackrox/stackrox/pull/5422/commits/56b7b0dd2efc0249d6df4cc04536c448da0d556c)